packages/nimble-components/src/table-column/table-column-text.ts

```diff
@@ -0,0 +1,81 @@
+import { attr, css, html, ViewTemplate } from '@microsoft/fast-element';
+import { DesignSystem } from '@microsoft/fast-foundation';
+import type { StringField, TableCellState } from '../table/types';
+import { TableColumn } from './table-column';
+import {
+    bodyFontColor,
+    bodyFont,
+    controlLabelFontColor
+} from '../theme-provider/design-tokens';
+
+type TableColumnTextCellData = StringField<'value'>;
+interface TableColumnTextColumnConfig {
+    placeholder: string;
+}
+
+/**
+ * The table column for displaying strings.
```

Comment:
After discussing with @rajsite we've decided to maintain the rigid display characteristics of this column as the guidance will be for clients to use the appropriate column type for any primitive value, as it will be likely that that the default display for non-string types for this column will not be what the user wants anyway, without them doing their own pre-formatting, in which case they can just convert to a string if desired at that point anyway.

Reply:
The other concern is not getting too loose with the type definition. Each column should be homogenous with one of the primitive types or null (following data table / pandas / column store / perspective / arrow type schemas). This was allowing something like `[{a: 'hi'}, {a: 2}]` which I think we should push back on. Like mentioned if the user wants to format it into a string, let them decide how to serialize it.